src/operators.jl

```diff
@@ -1,8 +1,8 @@
 ## Lifted operators
 
-importall Base.Operators
-import Base: promote_op, abs, abs2, sqrt, cbrt, scalarmin, scalarmax, isless
-using Compat: @compat
+import Base: !, %, &, *, +, -, /, //, <, <<, <<<, ==, ===, >, >>, >>>, \, ^, getindex, |, ~,
```

Comment:
I was mostly just guessing as to which operators need to be imported. Some of these are probably not necessary.